docs/source/install.rst

```diff
@@ -2,94 +2,97 @@ Dask Installation
 =================
 
 .. meta::
-    :description: Dask Installation | You can install Dask with conda, pip install Dask, or install from source.
+    :description: Dask Installation | You can easily install Dask with conda or pip
 
 .. _Anaconda distribution: https://www.anaconda.com/download/
 
-Dask currently supports Linux, macOS, and Windows. See the :doc:`changelog`
-for comprehensive release notes for each Dask version.
-
 How to Install Dask
 -------------------
 
-Dask installation can happen in a few different ways. You can install Dask with ``conda``, with ``pip``, or install from source.
+You can install Dask with ``conda``, with ``pip``, or install from source.
 
-Conda
------
+.. tab-set::
 
-If you use the `Anaconda distribution`_, Dask installation will occur by default.
-You can also install or upgrade Dask using the
-`conda install <https://docs.conda.io/projects/conda/en/latest/commands/install.html>`_ command::
+   .. tab-item:: Conda
 
-   conda install dask
+      If you use the `Anaconda distribution`_, Dask will be installed by default.
 
-This installs Dask and **all** common dependencies, including pandas and NumPy.
-Dask packages are maintained both on the default channel and on `conda-forge <https://conda-forge.github.io/>`_.
-You can select the channel with the ``-c`` flag::
+      You can also install or upgrade Dask using the
+      `conda install <https://docs.conda.io/projects/conda/en/latest/commands/install.html>`_ command::
 
-    conda install dask -c conda-forge
+         conda install dask
 
-Optionally, you can obtain a minimal Dask installation using the following command::
+      This installs Dask and **all** common dependencies, including pandas and NumPy.
+      Dask packages are maintained both on the defaults channel and on
+      `conda-forge <https://conda-forge.github.io/>`_.
+      You can select the channel with the ``-c`` flag::
 
-   conda install dask-core
+         conda install dask -c conda-forge
 
-This will install a minimal set of dependencies required to run Dask similar to (but not exactly the same as) ``python -m pip install dask`` below.
+      Optionally, you can obtain a minimal Dask installation using the following command::
 
-Pip
----
+         conda install dask-core
 
-You can use pip to install everything required for most common uses of Dask
-(e.g. Dask Array, Dask DataFrame, etc.).
-This installs both Dask and dependencies, like NumPy and pandas,
-that are necessary for different workloads. This is often the right
-choice for Dask users::
+      This will install a minimal set of dependencies required to run Dask similar to (but not exactly the same as) ``python -m pip install dask`` below.
```

Comment:
"Below" here felt a little off because of the tabs. Can probably just remove "below" altogether and not loose any meaning. 

```suggestion
      This will install a minimal set of dependencies required to run Dask similar to (but not exactly the same as) ``python -m pip install dask``.
```